eq-author-api/schema/tests/subscription.test.js

```diff
@@ -157,6 +175,64 @@ describe("subscriptions", () => {
         title: "",
       });
 
+      const p = iterator.next();
```

Comment:
Extremely small thing, feel free to ignore, but could this variable be a little more descriptive? Personally, I'm not a big fan of one letter variables.

Reply:
I have renamed to `iteratorNext`